proto/autoscaling/kubernetes/autoscaling.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+option go_package = "github.com/DataDog/agent-payload/v5/autoscaling/kubernetes";
+
+package datadog.autoscaling.kubernetes;
+
+// WorkloadValuesList represents a list of workload values
+message WorkloadValuesList {
+  repeated WorkloadValues values  = 1;
+}
+
+// WorkloadValues represents the values of a workload
+message WorkloadValues {  
+  string                   namespace = 1; // Namespace is the namespace of PodAutoscaler object
+  string                   name      = 2; // Name is the name of the PodAutoscaler object
+
+  WorkloadHorizontalValues horizontal = 3;
+  WorkloadVerticalValues   vertical   = 4;
+}
+
+message WorkloadHorizontalValues {
+  WorkloadHorizontalData manual = 1; // Manual is the manual override values for the workload
+  WorkloadHorizontalData auto   = 2; // Auto is the periodic recommendation values for the workload
```

Comment:
❓ Why do we need different types for the auto recommendation or the manual override? Aren't we just applying one of them in the Agent ?

Reply:
The controller will decide to apply manual/auto based on if dryrun or not.